source/reference/verification/custom-manual.txt

```diff
@@ -0,0 +1,86 @@
+.. _c2c-custom-manual-verification:
+
+==================================
+Custom or Manual Verification Page
+==================================
```

Comment:
Is this a placeholder name?

"Page" shouldn't be in a page title. I think something like "Verifying Data with Custom Scripts" is clearer. If we make this a full-fledged procedure in a future ticket, note that we use imperatives for those, so it would become "Verify Data with a Custom Script"